R/notify_loggr.R

```diff
@@ -13,14 +13,18 @@ notify_loggr <- function(..., type = "other", muffled = FALSE)
   args <- list(...)
   if (inherits(args[[1L]], "condition")) {
     cond <- args[[1L]]
-  } else if (type == "error") {
+  } else {
+    message = unlist(args)
+    message = paste0(message[1:(length(message)-3)],collapse='')
```

Comment:
Can we be sure that `length(message) > 3` here? We should use consistent syntax, `<-` for assignment, and spacing in argument list and around the minus.